packages/webdriverio/src/commands/element/isExisting.ts

```diff
@@ -61,6 +61,13 @@ export default async function isExisting (this: WebdriverIO.Element) {
         )
     }
 
-    const command = this.isReactElement ? this.parent.react$$.bind(this.parent) : this.parent.$$.bind(this.parent)
-    return command(this.selector as string).then((res) => res.length > 0)
+    const command = this.isReactElement
+        ? this.parent.react$.bind(this.parent)
+        : this.parent.$.bind(this.parent)
+    this.elementId = (await command(this.selector as string)).elementId
```

Comment:
Does this throw an error if `$` or `react$` can't find an element? What would be the return value and why does `command(this.selector).then(...)` doesn't return an element?

Reply:
is not throw an error if can't find an element.
I got your point when can't find an element, I'm adding `undefined` to the elementId.

Reply:
Updated also added 2 tests to make sure that we return false if the element can\'t be found.